Index Page structure done

zanarkandlee opened the pull request:
-color added
-footer made sticky
-added several images
-added modal for forgotten password

zanarkandlee committed:
Index Page structure done

-color added
-footer made sticky
-added several images
-added modal for forgotten password

wilsontanwm commented on traitquestdev/css/style.css:
```diff
@@ -15,29 +15,40 @@
 #logo-image{max-height:60px;}
+#header-bg{background-image:url('../images/bg-img.jpg');background-size: 100% 100%;}
+#footer-bg{background-color:#666666;margin-bottom:40px;}
+#cyan-bg{background-color:#D5FFFF;}
```
any particular reason, the following few lines need to be in here? can reuse the class in color css?

zanarkandlee replied:
the **cyan and dark green** background can be used from color.css

-for some reason, if i change the **footer-bg** ID and add class into the div, the margin and color doesn't appear
-i tried adding grey-bg class, it doesn't apply the margin, but it applies the color
-if it's lightgrey-bg class, both margin and color doesn't apply

for **header-bg** ID, the image is only 1 time use i think, so i used ID for it.

wilsontanwm commented on traitquestdev/index.php:
```diff
@@ -16,29 +20,244 @@
 <script src="//cdnjs.cloudflare.com/ajax/libs/jquery/1.7/jquery.min.js"></script>
 <script type="text/javascript" src="js/contact.js"></script>
+<script type="text/javascript" src="js/bootstrap.js"></script>
 
```
add a javascript so that when you click on the menu in the header, it will slide down to the respective section ie. features, testimonial etc

wilsontanwm commented on traitquestdev/index.php:
```diff
@@ -16,29 +20,244 @@
+		</div>
+
+		<div class="padding-topbottom-m">
+			<button class="registerButton">Register For Free</button>
```
thought you wanted to add the register form in the modal as well?

zanarkandlee committed:
Update 1 on index page frontend

-added several colors in color.css
-changed ID cyan-bg and darkgreen-bg to class
-ID header-bg and footer-bg not changed as it doesn't work after
changing to class
-register form changed to modal
-hidden testimonial from navbar
-anchor in navbar made smooth scrolling
-added smoothscroll.js for smooth scrolling to div ID

zanarkandlee committed:
Update 2 for index page frontend

-added mobile side navigation
-added smoothscroll for mobile nav
-added mobilenav.js
